src/components/DashboardPeriodSelector.vue

```diff
@@ -99,13 +100,15 @@ export default {
       immediate: true,
       async handler() {
         this.range = [this.startDate, this.endDate];
+        this.setSelectedPeriod(this.range);
       },
     },
 
     endDate: {
       immediate: true,
       async handler() {
         this.range = [this.startDate, this.endDate];
+        this.setSelectedPeriod(this.range);
```

Comment:
Holder det å kalle `this.setSelectedPeriod(range)` i watcheren for `range` i stedet (innenfor `if`-sjekken?). Unngår kanskje at den kalles unødig?